Examples/Nearest_Neighbour_Distance.ipynb

```diff
@@ -31,7 +31,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    var_search = var_search.search(path=var_search.df['path'][0])
</pre><p>What does this do/why we need it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>This is hack to only load this data from one file (it doesn't change in time, but is saved multiple time in the model output)</p><p> </p><p>Its equivalent to the <code>n=1</code> argument supplied to <code>getvar</code> in the cookbook version  </p>